pkg/config/load.go

```diff
@@ -83,7 +83,7 @@ func LoadTopicsFile(path string) ([]TopicConfig, error) {
 func LoadTopicBytes(contents []byte) (TopicConfig, error) {
 	config := TopicConfig{}
 	err := unmarshalYAMLStrict(contents, &config)
-	fmt.Println(config)
+	log.Infof("Loading config %+v", config)
```

Comment:
This was most likely included by mistake in topicctl, converted the print to a log line just in case